chart/kubeapps/templates/oauth2-secret.yaml

```diff
@@ -0,0 +1,15 @@
+{{- if .Values.authProxy.enabled }}
+apiVersion: v1
+kind: Secret
+metadata:
+  name: {{ template "kubeapps.oauth2_proxy-secret.name" . }}
+  {{- if .namespace }}
+  namespace: {{ .namespace }}
+  {{- end }}
+  labels:{{ include "kubeapps.labels" $ | nindent 4 }}
+data:
+  clientID: {{ required "You must fill \".Values.authProxy.clientID\" with the Client ID of the provider" .Values.authProxy.clientID | b64enc }}
+  clientSecret: {{ required "You must fill \".Values.authProxy.clientSecret\" with the Client Secret of the provider" .Values.authProxy.clientSecret | b64enc }}
+  cookieSecret: {{ required "You must fill \".Values.authProxy.cookieSecret\" with a 16, 24 or 32 byte base64 encoded seed string for secure cookies" .Values.authProxy.cookieSecret | b64enc }}
```

Comment:
Even if this message is clear, I'd add some example on how to generate this seed, such as `echo "not-good-secret" | base64`

Reply:
:+1: that's documented in a values.yaml comment but we can add that here too

Reply:
This have actually improved since that string was created. I don't think auth_proxy requires a specifically sized byte-string any more. Though they still use 16 in their own example. Might be good to point to the docs instead: https://oauth2-proxy.github.io/oauth2-proxy/docs/configuration/overview/#generating-a-cookie-secret

Reply:
:+1: (actually `not-good-secret` length is 15).

Reply:
It'd be great to also update the example in the values.yaml to just point to the same link.